arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-tree-widget.tsx

```diff
@@ -65,6 +74,28 @@ export class SketchbookTreeWidget extends FileTreeWidget {
         </React.Fragment>
     }
 
+    /**
+     * Render the node given the tree node and node properties.
+     * @param node the tree node.
+     * @param props the node properties.
+     */
+    protected renderNode(node: TreeNode, props: NodeProps): React.ReactNode {
```

Comment:
Why do we need this?

Reply:
we probably don't, let me remove it, thanks